benchcab/benchcab.py

```diff
@@ -265,32 +280,37 @@ def fluxsite_bitwise_cmp(self, config_path: str, verbose: bool):
         tasks = self.tasks if self.tasks else self._initialise_tasks(config)
         comparisons = get_fluxsite_comparisons(tasks)
 
-        print("Running comparison tasks...")
-        if config["fluxsite"]["multiprocess"]:
-            ncpus = config["fluxsite"]["pbs"]["ncpus"]
-            run_comparisons_in_parallel(comparisons, n_processes=ncpus, verbose=verbose)
+        self.logger.debug("Running comparison tasks...")
+        try:
+            multiprocess = config["fluxsite"]["multiprocess"]
+        except KeyError:
+            multiprocess = internal.FLUXSITE_DEFAULT_MULTIPROCESS
+        if multiprocess:
+            try:
+                ncpus = config["fluxsite"]["pbs"]["ncpus"]
+            except KeyError:
+                ncpus = internal.FLUXSITE_DEFAULT_PBS["ncpus"]
```

Comment:
Old version. We want:

```suggestion
        self.logger.debug("Running comparison tasks...")
         if config["fluxsite"]["multiprocess"]:
            ncpus = config["fluxsite"]["pbs"]["ncpus"]       
```

Reply:
Hi @ccarouge, I've committed the fix after running all tests again. Passing.

One thing to note, many of these remaining changes did not trigger failures in the tests. We may need to take a look at how robust our testing is to catch these in future.